dev-libs/libofx/libofx-0.9.14.ebuild

```diff
@@ -15,16 +15,23 @@ KEYWORDS="amd64 ~hppa x86"
 IUSE="static-libs test"
 
 RDEPEND="
+	>=dev-cpp/libxmlpp-2.40.1:2.6
 	>=net-misc/curl-7.9.7
 	virtual/libiconv
 "
 DEPEND="${RDEPEND}
 	>app-text/opensp-1.5
-	>=dev-cpp/libxmlpp-2.40.1:2.6
 	dev-util/gengetopt
+	test? ( app-crypt/gnupg )
+"
+BDEPEND="
 	sys-apps/help2man
 	virtual/pkgconfig
-	test? ( app-crypt/gnupg )"
```

Comment:
`app-crypt/gnupg` provides no headers and no libraries, but only tools, so it most likely should be in `BDEPEND`, not `DEPEND`.

Reply:
Wouldn't the same be true then for `dev-util/gengetopt`? It also has no headers or libraries, but only one binary.